lib/widgets/explore/product_item.dart

```diff
@@ -1,15 +1,57 @@
 import 'package:flutter/material.dart';
 
 class ProductItem extends StatelessWidget {
-  String title;
+  // TODO: Image;
+  final String title;
+  final String brand;
+  final String size; //? Should this be int or String?
```

Comment:
Could also be enum for XS, S, M etc or maybe a class for different types of sizes but I would say that a string is fine for now so remove the comment.
What do you think @emmaekst?

Reply:
Remove these comments.